index.html

```diff
@@ -10,951 +12,1067 @@
-            <div class="media-left">
-              <i class="fab fa-ravelry fa-4x"></i>
-            </div>
-            <div class="media-content">
-              <div class="content">
-                <p>
-                  <a><b>Avery Ipsum</b></a>
-                  <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean efficitur sit amet massa fringilla egestas. Nullam condimentum luctus turpis.
-                </p>
-              </div>
-            </div>
-          </div>
-          <div class="media">
-            <div class="media-left">
-              <i class="fab fa-github-alt fa-4x"></i>
-            </div>
-            <div class="media-content">
-              <div class="content">
-                <p>
-                  <a><b>io Ipsum</b></a>
-                  <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean efficitur sit amet massa fringilla egestas. Nullam condimentum luctus turpis.
-                </p>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
+			<div class="tab-pane" id="pane-4" role="tabpanel" aria-expanded="false">
+				<div class="columns is-centered">
+					<div class="column is-three-quarters">
+						<div class="title has-text-centered">Inspired video</div>
+						<div class="embed-container image">
+							<iframe src="https://player.vimeo.com/video/261794608" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
 
-    <div class="tab-pane" id="pane-4">
-      <div class="columns is-centered">
-        <div class="column is-three-quarters">
-          <div class="title has-text-centered">Inspired video</div>
-          <div class="embed-container image">
-            <iframe src="https://player.vimeo.com/video/261794608" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
+		<section class="section has-background-grey-lighter" aria-label="Upcoming events">
+			<div class="container has-text-centered">
+					<div class="section-heading has-text-centered">
+						<h2 class="title is-2">Upcoming events</h2>
+						<h3 class="subtitle is-5">Outcoming dates for the best options</h3>
+					</div>
 
-  <div class="section has-background-grey-lighter">
-    <div class="container has-text-centered">
-        <div class="section-heading has-text-centered">
-          <div class="title is-2">Upcoming events</div>
-          <div class="subtitle is-5">Outcoming dates for the best options</div>
-        </div>
+				<div class="tile is-ancestor">
+					<div class="tile is-parent">
+						<div class="tile is-child box">
+							<div class="image">
+								<img alt="image web access summit logo" src="./img/web-access-summit-eventbrite.jpg">
+							</div>
+							<h4 class="title" role="heading" aria-level="4">New York</h4>
```

Comment:
The `role="heading"` is redundant for this tag **<h4**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role

<img width="405" alt="image" src="https://user-images.githubusercontent.com/16785774/172357633-6ffa7ab8-0afe-4240-9d59-0d166aefed73.png">